share/shwap/p2p/bitswap/sample_block.go

```diff
@@ -71,6 +74,12 @@ func (sb *SampleBlock) Height() uint64 {
 	return sb.ID.Height
 }
 
+func (sb *SampleBlock) Size(ctx context.Context, acc eds.Accessor) (int, error) {
+	squareSize := acc.Size(ctx)
+	sampleSize := libshare.ShareSize + share.AxisRootSize*int(math.Log2(float64(squareSize)))
```

Comment:
is there no helper util we can use to determine proof size for a share based on dah?

Reply:
I am not aware of such